packages/app-settings/src/General.tsx

```diff
@@ -30,9 +30,10 @@ export default function General ({ className, isModalContent, onClose }: Props):
   const iconOptions = useMemo((): Option[] => {
     return uiSettings.availableIcons.map((o): Option => createIdenticon(t, o, ['default']));
   }, [t]);
-  const prefixOptions = useMemo((): Option[] => {
-    return uiSettings.availablePrefixes.map((o): Option => createOption(t, o, ['default']));
```

Comment:
Since we comment createOption, we may need to remove it from import for build to pass...